src/components/Pagination/Pagination.tsx

```diff
@@ -178,7 +178,9 @@ export const Pagination = ({
   const secondPage = decideSecondPage({ page, pages });
   const thirdPage = decideThirdPage({ page, pages });
   const forthPage = decideForthPage({ page, pages });
-  const showLastPage = page === pages - 2;
+  const showThirdPage = pages > 2;
+  const showForthPage = pages > 3;
+  const showLastPage = page < pages - 1;
```

Comment:
same with `page`, possibly `currentPage`?

Reply:
Why is my comment not loading? Github?

Reply:
I implemented all the other naming changes and it really helps, the code is a lot easier but when I put this one in things got a bit wordy:

```
        {showBackButton && <Back selectedPage={selectedPage} setSelectedPage={setSelectedPage} />}
        <PageButton selectedPage={1} setSelectedPage={setSelectedPage} isSelected={selectedPage === 1} />
        {showFirstElipsis && <div className={elipsisStyles}>...</div>}
        <PageButton
          selectedPage={secondPage}
          setSelectedPage={setSelectedPage}
          isSelected={selectedPage === secondPage}
        />
```

Same but using 'current'

```
        {showBackButton && <Back currentPage={currentPage} setCurrentPage={setCurrentPage} />}
        <PageButton currentPage={1} setCurrentPage={setCurrentPage} isSelected={currentPage === 1} />
        {showFirstElipsis && <div className={elipsisStyles}>...</div>}
        <PageButton
          currentPage={secondPage}
          setCurrentPage={setCurrentPage}
          isSelected={currentPage === secondPage}
        />
```

Or with no new changes:

```
        {showBackButton && <Back page={page} setPage={setPage} />}
        <PageButton page={1} setPage={setPage} isSelected={page === 1} />
        {showFirstElipsis && <div className={elipsisStyles}>...</div>}
        <PageButton
          page={secondPage}
          setPage={setPage}
          isSelected={page === secondPage}
        />
```

What do you think? 🤷‍♂

Reply:
I like `currentPage` and `isSelected` personally, makes it really easy to comprehend